src/main/java/MeetingOrganizer.java

```diff
@@ -0,0 +1,172 @@
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
```

Comment:
I felt that section 3.2, Design Considerations was very useful. It takes into account the profile of an experienced developer, but is easy enough to understand for a new developer.